docs/source/intervention_models/targeted_bep_and_mms/index.rst

```diff
@@ -171,71 +171,115 @@ For the implementation of the intervention in an alterative scenarios, we will m
 Birthweight
 +++++++++++++++++++++
 
-.. important::
+The maternal supplementation intervention (all regimens) affect child birthweight exposures, :ref:`which are documented here <2019_risk_exposure_lbwsg>`. The intervention should result in an **additive change to a simulant's continuous birthweight exposure value at birth (or upon initialization into the early or late neonatal age groups).** We assume there is no corresponding change in a simulant's gestational age exposure value at birth.
 
-  Copy and paste this section for each affected outcome included in this document
-
-.. todo::
-
-  Replace "Risk Outcome Pair #1" with the name of an affected entity for which a modeling strategy will be detailed. For additional risk outcome pairs, copy this section as many times as necessary and update the titles accordingly.
-
-.. todo::
-
-  Link to existing document of the affected outcome (ex: cause or risk exposure model document)
-
-.. todo::
-
-  Describe exactly what measure the intervention will affect
-
-.. todo::
-
-  Fill out the tables below
-
-.. list-table:: Affected Outcome #1 Restrictions
-  :widths: 15 15 15
+.. list-table:: Restrictions for intervention effect on birthweight
   :header-rows: 1
 
   * - Restriction
     - Value
     - Note
   * - Male only
-    - 
+    - False
     - 
   * - Female only
-    - 
+    - False
     - 
   * - Age group start
-    - 
+    - Birth
     - 
   * - Age group end
-    - 
+    - Late neonatal
     - 
   * - Other
     - 
     - 
 
-.. list-table:: Affected Outcome #1 Effect Size
-  :widths: 15 15 15 
+.. list-table:: Supplementation effect on birthweight
   :header-rows: 1
 
   * - Population
-    - Effect size
+    - Effect size (95% CI)
     - Note
-  * - 
-    - 
-    - 
-  * - 
+    - Source
+  * - Unsupplemented mothers
+    - 0
     - 
-    - 
-
-.. todo::
-
-  Describe exactly *how* to apply the effect sizes to the affected measures documented above
+    - N/A
+  * - IFA supplemented mothers (overall)
+    - +57.73 g (7.66 to 107.79)
+    - Relative to no supplementation 
+    - [Pena-Rosas-et-al-2015]_
+  * - MMN supplemented mothers (overall)
+    - +45.16 (32.31 to 58.02) 
+    - Relative to IFA
+    - Meta-analysis of 13 trials from [Keats-et-al-2019-maternal-supplementation]_, linked as a memo in :ref:`BEP concept model document <2017_concept_model_vivarium_gates_bep>`
+  * - BEP supplemented mothers (undernourished)
+    - +66.96g (13.13 to 120.78)
+    - Relative to MMN
+    - [Ota-et-al-2015]_
```

Comment:
@alibow Is that means the birthweight shift = +57.73g if someone takes IFA; and +(57.73 + 45.16)g for MMN, and +(57.73 + 45.16 + 66.96)g for BEP; assuming all three groups compare to no supplementations.

Reply:
Yep, exactly!

Reply:
@yongqx, can you please approve this PR if you have finished reviewing? I can merge all of the supplementation PRs after this one is approved. Thanks!

Reply:
Thank you!